examples/stable_diffusion/convert_sd_adapter_to_peft.py

```diff
@@ -202,6 +203,11 @@ def detect_adapter_type(keys: List[str]) -> PeftType:
     parser.add_argument("--dump_path", default=None, type=str, required=True, help="Path to the output peft adapter.")
 
     parser.add_argument("--half", action="store_true", help="Save weights in half precision.")
+    parser.add_argument(
+        "--loha_weights_fix",
+        action="store_true",
+        help="Apply LoHa weights fix from this PR https://github.com/KohakuBlueleaf/LyCORIS/pull/115",
```

Comment:
I think the message should be expanded a little bit to make it very clear to the user when to use this flag: If the weights they use are created before the mentioned PR was merged, this argument should be True, if they're from after the PR was merged, it should be False.